<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1074</width>
    <height>747</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>253</width>
      <height>371</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="minimumSize">
        <size>
         <width>251</width>
         <height>0</height>
        </size>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>-361</y>
          <width>232</width>
          <height>800</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>800</height>
         </size>
        </property>
        <widget class="QPushButton" name="pb_show_origin">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>211</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string>Show Origin </string>
         </property>
        </widget>
        <widget class="QPushButton" name="pb_grayscaleAB">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>410</y>
           <width>211</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string>Gray Scale A/B</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pb_diff_AB">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>470</y>
           <width>211</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string>Difference Between A/B </string>
         </property>
        </widget>
        <widget class="QPushButton" name="pb_binary">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>530</y>
           <width>121</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string>Binary Thresh</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pb_equalization">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>80</y>
           <width>211</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string>Historgram Equalization</string>
         </property>
        </widget>
        <widget class="QSlider" name="slider_brightness">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>190</y>
           <width>160</width>
           <height>18</height>
          </rect>
         </property>
         <property name="sliderPosition">
          <number>50</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>210</y>
           <width>101</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Brightness</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QSlider" name="slider_contrast">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>250</y>
           <width>160</width>
           <height>18</height>
          </rect>
         </property>
         <property name="sliderPosition">
          <number>50</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>150</y>
           <width>101</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Contrast</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="label_10">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>280</y>
           <width>101</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Enlarge</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit">
         <property name="geometry">
          <rect>
           <x>12</x>
           <y>533</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>570</y>
           <width>58</width>
           <height>15</height>
          </rect>
         </property>
         <property name="text">
          <string>0-255</string>
         </property>
        </widget>
        <widget class="QDoubleSpinBox" name="doubleSpinBox">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>290</y>
           <width>91</width>
           <height>31</height>
          </rect>
         </property>
         <property name="minimum">
          <double>1.000000000000000</double>
         </property>
         <property name="maximum">
          <double>10.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
        <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>340</y>
           <width>91</width>
           <height>31</height>
          </rect>
         </property>
         <property name="decimals">
          <number>1</number>
         </property>
         <property name="minimum">
          <double>0.100000000000000</double>
         </property>
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="stepType">
          <enum>QAbstractSpinBox::DefaultStepType</enum>
         </property>
        </widget>
        <widget class="QLabel" name="label_11">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>330</y>
           <width>101</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Shrink</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="label_12">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>590</y>
           <width>101</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Graylevel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>600</y>
           <width>91</width>
           <height>31</height>
          </rect>
         </property>
         <property name="minimum">
          <double>2.000000000000000</double>
         </property>
         <property name="maximum">
          <double>256.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>2.000000000000000</double>
         </property>
         <property name="value">
          <double>4.000000000000000</double>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>440</y>
      <width>351</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="histo_1">
     <item>
      <widget class="QLabel" name="lb_chart_a">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="lb_img_a">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>70</y>
      <width>351</width>
      <height>351</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lb_img_b">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>70</y>
      <width>351</width>
      <height>351</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Gray Scale : A</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>30</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Gray Scale : B</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>440</y>
      <width>351</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="histo_2">
     <item>
      <widget class="QLabel" name="lb_char_b">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>251</width>
      <height>242</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QScrollArea" name="scrollArea_2">
       <property name="minimumSize">
        <size>
         <width>240</width>
         <height>240</height>
        </size>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>300</width>
          <height>300</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>300</height>
         </size>
        </property>
        <widget class="QLabel" name="lb_oringimg">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>221</width>
           <height>221</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1074</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
